src/main/java/games/strategy/engine/framework/startup/ui/InGameLobbyWatcher.java

```diff
@@ -1,5 +1,12 @@
 package games.strategy.engine.framework.startup.ui;
 
+import static games.strategy.engine.framework.ArgParser.CliProperties.LOBBY_GAME_COMMENTS;
+import static games.strategy.engine.framework.ArgParser.CliProperties.LOBBY_GAME_HOSTED_BY;
+import static games.strategy.engine.framework.ArgParser.CliProperties.LOBBY_HOST;
+import static games.strategy.engine.framework.ArgParser.CliProperties.LOBBY_PORT;
+import static games.strategy.engine.framework.ArgParser.CliProperties.TRIPLEA_PORT;
+import static games.strategy.engine.framework.ArgParser.CliProperties.SERVER_PASSWORD;
```

Comment:
Please reorder the imports in this file.  Checkstyle is reporting they are not in lexicographical order.